### DISCUSSION TITLE
Include Attribute Editor with global fields

### DISCUSSION THREAD

**Discussion by chrismlusk** [solspace/craft-freeform#503] [Feature Requests]
It would be nice if the Attribute Editor setting was available when creating global fields so that we could apply attributes that are attached by default any time a user places that field in a form.

**Comment by kjmartens** [comment]
This has now been implemented in Freeform 5. Using the Favorite Fields feature, you can set any field to be "global" and assign custom attributes to them, etc. 🙂
